client/js/tracking.js

```diff
@@ -0,0 +1,16 @@
+import { on, nodeList } from './util';
+import { trackEvent } from './utils/track-event';
+
+// Transporter
+on('body', 'click', '[data-track="transporter"]', (event) => {
+  const transporter = event.target.closest('[data-track="transporter"]');
+  const promos = nodeList(transporter.querySelectorAll('.promo'));
+  const promo = event.target.closest('.promo');
+  const link = promo ? promo.href : null;
+  const position = promo ? promos.indexOf(promo) : null;
+  const properties = {id: transporter.id, link, position};
+
+  if (link) {
+    trackEvent({name: 'Transporter', properties});
+  }
+});
```

Comment:
Are you going to give something like this a go?
Just wondering if I wait on that before reviewing - if so, maybe mark the PR as WIP.